run.sh

```diff
@@ -59,6 +60,20 @@ done
 # Delete any remaining empty dirs
 find target/ -type d -empty -delete
 
+# Kick off Glue crawler run. Not strictly necessary since 99%
+# of the time we're not creating new partitions or columns,
+# but still nice to run
+/usr/bin/aws glue \
+    start-crawler --name "$CRAWLER_NAME" \
+    | ts '%.s' \
+    | tee -a "$TEMP_LOG_FILE"
```

Comment:
Basically only running this in case the scheduled crawler run is mis-timed i.e. runs before the extraction completes.